### ISSUE TITLE
makefile troubles

### ISSUE THREAD

**Issue by floswald** [kurtmitman/housing-boom-bust#1]
hi!

sorry to bother with this, but I get an error in the makefile. 

```
➜  git git clone https://github.com/kurtmitman/housing-boom-bust.git KMV
Cloning into 'KMV'...
remote: Enumerating objects: 195, done.
remote: Counting objects: 100% (195/195), done.
remote: Compressing objects: 100% (114/114), done.
remote: Total 195 (delta 80), reused 191 (delta 79), pack-reused 0
Receiving objects: 100% (195/195), 35.18 MiB | 16.82 MiB/s, done.
Resolving deltas: 100% (80/80), done.
➜  git cd KMV
➜  KMV git:(master) ls
Data        FortranCode LICENSE     MatlabCode  README.md   Reamde.docx StataCode   scratch
➜  KMV git:(master) cd FortranCode 
➜  FortranCode git:(master) ls
Makefile          benchmark-code    landlord-code     rental-wedge-code sensitivity-code
➜  FortranCode git:(master) make
make: *** No rule to make target `params.o', needed by `globals.o'.  Stop.
```

did i miss something here to compile this? I tried to find other targets in the Makefile but couldn't (I mean, `make clean` ran ok, but that's not really what I want now ;-) )

**Comment by kurtmitman** [comment]
Hi Florian - sorry, this was sloppy on my part.

The Makefile included works with each of the four versions of the code. But since I figured that people would need to edit the Makefile to match the specs on their system, I only included one copy of it. I'll edit the readme. Basically, copy the makefile into the subdirectory that you would like to build and then run make. Hopefully, that shouldn't provide any problems.
